lib/plugins/features/movies/index.js

```diff
@@ -32,6 +33,19 @@ exports.register = (server, options, next) => {
     }
   }]);
 
+  server.route([{
```

Comment:
I just realized this so it's not a big deal and definitely doesn't need to necessarily be fixed, but all of these endpoints can be moved into a single array into a single `server.route` call.